cluster/node-pools/worker-default/userdata.clc.yaml

```diff
@@ -229,7 +229,11 @@ systemd:
       --cpu-cfs-quota=false \
 {{- end }}
       --system-reserved=cpu=100m,memory=164Mi \
-      --kube-reserved=cpu=100m,memory=282Mi
+      --kube-reserved=cpu=100m,memory=282Mi \
+      --client-ca-file=/etc/kubernetes/ssl/ca.pem \
+      --anonymous-auth=false \
+      --authentication-token-webhook=true \
+      --authorization-mode=Webhook
```

Comment:
maybe you do not need the webhook flags on worker node kubelets?

Reply:
This is needed for kubelet to make SubjectAccessReview calls to the API server authorize requests. See: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-authentication-authorization/#kubelet-authorization

Reply:
Interesting, as far as I understand it's used for the token authentication/authorization. So requests that go to kubelet with a token auth, will hit the apiserver webhook passed by the kubelet.
It would be nice if a flow chart or something like this would exist in the k8s docs for this. It would make it much more clear.

Reply:
> Interesting, as far as I understand it's used for the token authentication/authorization. So requests that go to kubelet with a token auth, will hit the apiserver webhook passed by the kubelet.

Correct! Thanks for creating the issue upstream :)